### ISSUE TITLE
Create an API

### ISSUE THREAD

**Issue by AdamFinkle** [sfbrigade/datasci-earthquake#19]
## Context
The front end will need to call the back end to obtain data and potentially calculations.  

## Definition of Done
An API with an exposed method for every back end need of the front end.

## Engineering Details
Solving this issue requires asking the front end what they need from the back end and then clarifying the answers into methods to call.  It will also involve creating a file or files containing the API.

**Comment by mackcooper1408** [comment]
@AdamFinkle let's break this up into smaller tasks. I've laid out a bunch of config tickets to get the backend set up, so I think this issue can be split up into small, directly actionable tasks like "create API endpoint for getting liquefaction layers in a given region" with specific details about the requirements and details on what it's expected to receive from the client and return back to the client.

I also think we need a "SPIKE" ticket (basically just a research/documentation) ticket that blocks API creation tasks and actually creates a formal, approved documentation for what we want our API to generally _do_ and the different endpoints we want to expose. _**Then**_, we can use that documentation to create separate issues for each endpoint and can even couple these issue with corresponding Frontend features.

TLDR - as it stands, this issue is too vague to implement at this time

**Comment by mackcooper1408** [comment]
In general, this is handled, closing this
